Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -271,8 +272,11 @@ def validateInputs(self):
 
         # If doing absorption correction, make sure the sample formula is correct
         if self.getProperty("TypeOfCorrection").value != "None":
-            if self.getProperty("SampleFormula").value == '':
-                issues['SampleFormula'] = "A sample formula must be provided."
+            if self.getProperty("SampleFormula").value != '':
```

Comment:
`strip()` the value first. Various people will accidentally leave a bit of whitespace in the GUI when turning off previously set properties